src/components/ListedSpecies/BehaviorsSection.jsx

```diff
@@ -44,16 +47,23 @@ const BehaviorsSection = ({ behaviorOptions, nestingOptions }) => {
                 closeMenuOnSelect={false}
               />
             </FormLabel>
+            {isTemplate && <Text color="#718096">Static</Text>}
           </FormControl>
         </GridItem>
       </Grid>
     </CollapsibleSection>
   );
 };
 
+BehaviorsSection.defaultProps = {
+  isTemplate: false,
+};
 BehaviorsSection.propTypes = {
   behaviorOptions: PropTypes.arrayOf(PropTypes.string).isRequired,
   nestingOptions: PropTypes.arrayOf(PropTypes.string).isRequired,
+  nesting: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
🚫 **[eslint]** <[react/no-unused-prop-types](https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/no-unused-prop-types.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'nesting' PropType is defined but prop is never used